### ISSUE TITLE
Project - Painting feature

### ISSUE THREAD

**Issue by Garanas** [FAForever/fa#5976]
# Painting 

## Description

At the moment we have a painting feature for observers. By painting we only mean the ability to draw lines. The feature allows an observer to paint on the map and other observers can view the painting. 

![image](https://github.com/FAForever/fa/assets/15778155/a66320e0-5670-4a76-912f-ca7ab62a5f21)

This feature can be quite powerful and we wish to extend it. In particular we want regular players to be able to paint, but we also want to improve the painting itself. 

## Tasks

- [ ] Interpolation I
- [ ] Interpolation II
- [ ] Update frequency
- [ ] Message in chat

**Comment by GodFuper** [comment]
When development #6726 is completed. Then can use it code to make a cursor display for your allies, which will work not only in the game, but also in replay. The mechanism will be similar.

Now there is a similar mechanism called "show mouse locations of players", but it is only for the observer.

**Comment by 4z0t** [comment]
Replays will significantly grow in size. In addition this will add data to be passed through the net, which may make even worse current state of connection in games.

**Comment by Garanas** [comment]
I agree - storing the cursor in the replay is not desirable. I'm not sure if it will impact the state of connection in games. But in order for something to appear in a replay it needs to be a sim callback. All players receive all sim callbacks. Therefore if we want the cursor in the replay then technically all players know the cursor position of all other players.

That would be too interesting for cheaters in my opinion.
